### PR TITLE
Add new acceptance test case divider block [MAILPOET-3325]

### PR DESCRIPTION
**Pull request by veljkho**
[MAILPOET-3325](https://mailpoet.atlassian.net/browse/MAILPOET-3325)

### CODE REVIEW

File: tests/acceptance/FormEditorAddDividerCest.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    $i->login();
+    $i->amOnMailPoetPage('Forms');
+    $i->waitForText($formName);
+    $i->clickItemRowActionByItemName($formName, 'Edit');
+    $i->waitForElement('[data-automation-id="form_title_input"]');
+
+    // Add divider & spacer block
+    $i->addFromBlockInEditor('Divider / Spacer');
+
+    // Check and modify divider & spacer block
```
Comment (costasovo) [inline tests/acceptance/FormEditorAddDividerCest.php:27]: I still think that this is a perfect example of comment that could be place into $i->wantTo function.

File: tests/acceptance/FormEditorAddDividerCest.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    $i->click('.mailpoet-automation-divider-togle-enable');
+    $i->selectOption('[data-automation-id="settings_divider_style"]', 'Dotted');
+    $i->seeInField('.mailpoet-automation-styles-divider-height input[type="number"]', 1);
+    $i->clearFormField('.mailpoet-automation-styles-divider-height input[type="number"]');
+    $i->fillField('.mailpoet-automation-styles-divider-height input[type="number"]', 10);
+    $i->seeInField('.mailpoet-automation-styles-divider-width input[type="number"]', 100);
+    $i->clearFormField('.mailpoet-automation-styles-divider-width input[type="number"]');
+    $i->fillField('.mailpoet-automation-styles-divider-width input[type="number"]', 10);
+
+    // Save and reload page to check if data were saved
```
Comment (costasovo) [inline tests/acceptance/FormEditorAddDividerCest.php:43]: I would use I->wantTo here as well.

**Review by costasovo — APPROVED** [review]
Looks good! I would just suggest replacing a couple of comments with I->wantTo. I approve and leave it to you to decide.

**Review by costasovo — APPROVED** [review]

File: tests/acceptance/FormEditorAddDividerCest.php (hunk 1 above)
Comment (veljkho) [reply tests/acceptance/FormEditorAddDividerCest.php]: Done.

**Review by costasovo — APPROVED** [review]

**Comment by costasovo** [comment]
Thanks!

**Pull request merged**